lib/bridge/IrcBridge.js

```diff
@@ -448,9 +448,23 @@ IrcBridge.prototype.getThirdPartyProtocol = function(protocol) {
     var servers = this.getServers();
 
     return Promise.resolve({
-        "user_fields": ["domain", "nick"],
-        "location_fields": ["domain", "channel"],
-        "instances": servers.map((server) => {
+        user_fields: ["domain", "nick"],
+        location_fields: ["domain", "channel"],
+        field_types: {
+            domain: {
+                regexp: "[a-z0-9-_]+(\.[a-z0-9-_]+)*",
+                placeholder: "irc.example.com",
+            },
+            nick: {
+                regexp: "[^#\\s]+",
+                placeholder: "SomeNick",
+            },
+            channel: {
+                regexp: "[#&+][^\\s]+",
```

Comment:
(The bridge doesn't really *do* `&chan` and `+chan` rooms, so I would cheekily request you remove that from the `regexp` :angel: )